modules/installation-and-upgrade/pages/container-deployment/suma/proxy-deployment-suma.adoc

```diff
@@ -53,23 +52,30 @@ For more information about hardware requirements for deploying {productname} Pro
 
 == Synchronize the Parent and Proxy Extension Child Channels
 
-This section presumes that you have already entered your organization credentials under the menu:Admin[Setup Wizard -> Organization Credentials] in the server's {webui}.
-Products are listed on the menu:Admin[Setup Wizard -> Products] page.
+This section presumes that you have already entered your organization credentials under the menu:Admin[Setup Wizard > Organization Credentials] in the server's {webui}.
+Products are listed on the menu:Admin[Setup Wizard > Products] page.
 This channel must be fully synchronized on the server, with the child channel [systemitem]``Proxy`` as an extension option selected.
 
 
 .Procedure: Synchronizing the Proxy Parent Channel and Proxy Extension
 . In the {productname} {webui} select menu:Admin[Products].
-. From the products page enter {sle-micro} in the filter field.
+. From the products page enter {sle-micro} or {sles} in the filter field.
 . Next use the drop-down to select the required architecture.
   For this example {x86_64}.
-. In the [systemitem]``Product Description`` field select the {sle-micro} {microversion} checkbox then use the drop-down to select the [systemitem]``SUSE Manager Proxy Extension 5.0 x86_64`` extension.
+. In the [systemitem]``Product Description`` field select the {sle-micro} {microversion} or {sles} {bci-suma} checkbox then use the drop-down to select the [systemitem]``SUSE Manager Proxy Extension 5.0 x86_64`` extension.
 . Click the btn:[Add products] button.
 . Wait for the synchronization to complete.
 
 
 
-== Deploy {productname} Proxy on {sle-micro} {microversion}
+[[deploy-suma-proxy-host]]
+==  Deploy {productname} Proxy
```

Comment:
I agree with the FIXME below.
This is all just about preparing the proxy host.

```suggestion
==  Prepare {productname} Proxy Host
```

and similarly the sub chapters.

Reply:
Thanks. I'll commit sth. from CL.  I can change it again, if we find a better phrase.